scripts/rv_expression_association/run_SKAT.R

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env Rscript
+
```

Comment:
Perhaps here you could just have a one-liner about the main purpose of the script

Reply:
will do, thanks!